src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.html

```diff
@@ -0,0 +1,47 @@
+<mat-toolbar color="primary" class="app-bar mat-elevation-z4" data-cy="pxb-toolbar">
+    <button data-cy="toolbar-menu" *ngIf="isSmall" style="margin-left: -8px" mat-icon-button (click)="toggleMenu()">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Phone Number Format</div>
+</mat-toolbar>
+<section class="mat-typography">
+    <form [formGroup]="validatePhoneNumberForm" class="phone-number-form">
+        <mat-form-field class="country-code-select" appearance="fill">
+            <mat-label>Country Code</mat-label>
+            <mat-select
+                formControlName="selectedCountry"
+                (selectionChange)="onCountryChange($event)"
+                panelClass="selectPanel"
+                [disableOptionCentering]="true"
+            >
+                <mat-option [value]="option.code" *ngFor="let option of countries">{{ option.name }}</mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field class="phone-number-input" appearance="fill">
+            <mat-label>Phone Number</mat-label>
+            <input
+                matInput
+                (blur)="validatePhoneNumber()"
+                formControlName="phone"
+                [value]="
+                    validatePhoneNumberForm.get('phone').value
+                        | formatPhoneNumber: selectedCountry['code']:countryChange
+                "
+                inputmode="numeric"
+                [maxlength]="selectedCountry['maxLength']"
+                (keypress)="allowNumbersOnly($event)"
+                (focus)="removeError()"
+                [placeholder]="selectedCountry['placeholder']"
+            />
+
+            <mat-error
+                *ngIf="
+                    validatePhoneNumberForm.get('phone').hasError('phoneNumberNotMatch') ||
```

Comment:
can we change this error key to something like "invalidFormat" or even as explicit as "invalidPhoneNumberFormat"?